src/main/java/com/exasol/errorcodecrawlermavenplugin/validation/AbstractIndependentErrorMessageDeclarationValidator.java

```diff
@@ -0,0 +1,31 @@
+package com.exasol.errorcodecrawlermavenplugin.validation;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+import com.exasol.errorcodecrawlermavenplugin.Finding;
+import com.exasol.errorcodecrawlermavenplugin.model.ErrorMessageDeclaration;
+
+/**
+ * Abstract basis for {@link ErrorMessageDeclarationValidator}s that validate each {@link ErrorMessageDeclaration}
+ * independent of the others.
+ */
+public abstract class AbstractIndependentErrorMessageDeclarationValidator implements ErrorMessageDeclarationValidator {
```

Comment:
please check if other classes might extends this one

Reply:
done